src/braket/pennylane_plugin/translation.py

```diff
@@ -414,6 +420,49 @@ def _(adjoint: Adjoint, parameters):
     return base.adjoint()[0]
 
 
+@_translate_operation.register
+def _(op: ParametrizedEvolution, _parameters, device):
+    start, end = op.t[0], op.t[1]
+    pulse_length = (end - start) * 1e-9  # nanoseconds to seconds
+    pulses = op.H.pulses
+
+    # The driven wires aren't the same as `op.wires` as `op.wires` contains
+    # all device wires due to interaction term.
+    op_wires = qml.wires.Wires.all_wires([pulse.wires for pulse in pulses])
+    mapped_wires = op_wires.map(device.wire_map)
+
+    frames = {w: device._device.frames[f"q{w}_drive"] for w in mapped_wires}
+    time_step = frames[0].port.dt * 1e9  # seconds to nanoseconds
+
+    pulse_sequence = PulseSequence().barrier(frames.values())
+
+    for pulse in pulses:
+        # Create waveform for each pulse in `ParametrizedEvolution`
+        if callable(pulse.amplitude):
+            amplitude = partial(pulse.amplitude, op.parameters[0])
+
+            # Calculate amplitude for each time step and normalize
+            amplitudes = onp.array(
+                [amplitude(t) for t in np.arange(start, end + time_step, time_step)]
+            )
+
+            waveform = ArbitraryWaveform(amplitudes)
+
+        else:
+            waveform = ConstantWaveform(pulse_length, amplitude)
+
+        # Play pulse for each frame
+        for w in pulse.wires.map(device.wire_map):
+            pulse_sequence = (
+                pulse_sequence.set_frequency(frames[w], pulse.frequency * 1e9)  # GHz to Hz
+                .set_phase(frames[w], pulse.phase)
+                .play(frames[w], waveform)
+            )
+
+    pulse_sequence = pulse_sequence.barrier(frames.values())
+    return gates.PulseGate(pulse_sequence, num_qubits=len(op_wires))
```

Comment:
```suggestion
    return gates.PulseGate(pulse_sequence, qubit_count=len(op_wires))
```

Reply:
Also because of things that happen in the plugin `braket_device.py` line 414 and in [this braket file](https://github.com/aws/amazon-braket-sdk-python/blob/main/src/braket/circuits/instruction.py), the `qubit_count` needs to be the same as the number of qubits on the device, no on the operator. Not sure why, but since its standard for the whole braket device, I don't really dare change it. Therefore:

```suggestion
    return gates.PulseGate(pulse_sequence, qubit_count=len(device.wires))
```